.rubocop.yml

```diff
@@ -46,23 +48,42 @@ Style/DotPosition:
 Style/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: true
 
+# Maintain compatibility with older Rubies
+Style/FrozenStringLiteralComment:
+  Enabled: false
+
 # Allow multiline block chains
 Style/MultilineBlockChain:
   Enabled: false
 
+# Indent one level for follow-up lines
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+# Indent one level for follow-up lines
+Style/MultilineOperationIndentation:
+  EnforcedStyle: indented
+
+# TODO: Discuss
+Style/MutableConstant:
+  Enabled: false
+
+# TODO: Discuss
+Style/ParallelAssignment:
+  Enabled: false
+
 # Allow Perl-style references to regex matches
 Style/PerlBackrefs:
   Enabled: false
 
-Style/Documentation:
-  Exclude:
-    - 'lib/reek/ast/sexp_extensions/send.rb'
-    - 'lib/reek/ast/sexp_extensions/super.rb'
-    - 'lib/reek/ast/sexp_extensions/variables.rb'
-
-Style/AccessorMethodName:
-  Exclude:
-    - 'lib/reek/context/visibility_tracker.rb'
+# Always use raise to signal exceptions
+Style/SignalException:
+  EnforcedStyle: only_raise
```

Comment:
Ok, I removed the setting so we use RuboCop's new default (which is `only_raise` so there are no code changes).